postman-tips.md

```diff
@@ -110,3 +128,50 @@ In hetzelfde [Parameters Object](https://github.com/OAI/OpenAPI-Specification/bl
                 ],
 
 ```
+
+## Standard apikey & tenant
+
+De meeste api's verwachten een variabele voor de apikey en de tenant in de header. Als we deze twee header variabele al toevoegen bij elke operatie, wordt het weeral wat makkelijker:
+
+```js
+    "paths": {
+        "/addons": {
+            "get": {
+                "description": "Get a list of addons that a user can select.",
+                "summary":"Find addons with an optional search query",
+                "parameters": [
+                    {
+                        "name": "apikey",
+                        "description": "from a contract in the api store",
+                        "in": "header",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "tenant",
+                        "description": "the name of the tenant you want to use",
+                        "in": "header",
+                        "required": true,
+                        "type": "string"
+                    }
+                ],
+
+```
+
+
+## Postman environment files
+
+als je de instellingen in de swagger zoals hierboven beschreven maakt, kan je volgende variabele in postman voorzien.
+
+### globale environment variabelen
+
+* definieer de `schema` globale variabele in postman met waarde `https`
+* definieer de `host`globale variabele met waarde `api-gw.antwerpen.be`
```

Comment:
gw url is omgevingsafhankelijk : api-gw-o.antwerpen.be, api-gw-a.antwerpen.be en api-gw-p.antwerpen.be